content/posts/sentakki/index.md

```diff
@@ -106,15 +105,16 @@ Sentakki is primarily designed for touch screen. However you can play it normall
 
 - ![Spin Icon](mod-icon/spin-mod.png) Spin
   - Replicate the true washing machine experience.
+    - Spins the playfield during gameplay
 
 - ![Experimental Icon](mod-icon/experimental-mod.png) Experimental
   - Some experimental features to be added to future sentakki builds. Autoplay/No-Fail recommended. Replay unsupported.
-  - Function that you can enable to experience it are *Twin notes*, *Twin slides* and *Touch notes*
+  - Experiment that are availables are *Twin notes*, *Twin slides* and *Touch notes*
 
 # settings
 
-You can see a full setting detail in [Sentakki option wiki page](https://github.com/LumpBloom7/sentakki/wiki/Options).
+You can see a list of settings and what they do in [Sentakki option wiki page](https://github.com/LumpBloom7/sentakki/wiki/Options).
 
 # demo
 
-{{< youtube id="DEpT1ReTKo0" >}}
\ No newline at end of file
+{{< youtube id="BDOc3PSDYFU" >}}
```

Comment:
<img width="787" alt="Screen Shot 2564-05-01 at 19 42 19" src="https://user-images.githubusercontent.com/68165621/116782786-595cf980-aab5-11eb-9315-840c5c4021c9.png">
I think you forget to change visibilty.

Reply:
Fixed :P